src/tracer/MOM_generic_tracer.F90

```diff
@@ -423,6 +433,33 @@ subroutine initialize_MOM_generic_tracer(restart, day, G, GV, US, h, tv, param_f
     enddo
     !! end section to re-initialize generic tracers
 
+    call get_param(param_file, "MOM", "PHA_MLD_FIXED", CS%mld_pha_fixed, &
+                 "If true, use a fixed value for the photoacclimation mixed layer depth within the "//&
+                 "generic tracer update. This MLD is only used for photoacclimation.", default=.true.)
+    if (CS%mld_pha_fixed) then
+      call get_param(param_file, "MOM", "PHA_MLD_VAL", CS%mld_pha_val, &
```

Comment:
We can change it to a more realistic value, 75m or whatever @charliestock recommends.

Reply:
@charliestock should chime in here. 
We discussed it last week and the choice of a "bad" default is because a constant value should not be used with COBALTv3. However, we need some value in the feild for non-COBALTv3 simulations that still se the generic tracer interface.